website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -316,6 +318,51 @@ This macro yields the database-specific data type for a `TIMESTAMP` (which may o
 TIMESTAMP
 ```
 
+### dbt.current_timestamp()
+
+The `dbt.current_timestamp()` macro allows you to generate the current timestamp in a format suitable for database queries. This is useful when you need to capture the current date and time within your SQL queries or models.
```

Comment:
@dbeatty10 is this the same for our other adapters? starburst, fabric, spark, databricks?